runtime/src/lib.rs

```diff
@@ -321,31 +360,45 @@ type LocalAssetTransactor = xcm_builder::CurrencyAdapter<
 type LocalOriginConverter = (
 	SovereignSignedViaLocation<LocationConverter, Origin>,
 	RelayChainAsNative<RelayChainOrigin, Origin>,
-	SiblingParachainAsNative<cumulus_pallet_xcm_handler::Origin, Origin>,
+	SiblingParachainAsNative<cumulus_pallet_xcm::Origin, Origin>,
 	SignedAccountId32AsNative<CentrifugeNetwork, Origin>,
 );
 
+pub type Barrier = (
+	TakeWeightCredit,
+	AllowTopLevelPaidExecutionFrom<All<MultiLocation>>,
+	AllowUnpaidExecutionFrom<All<MultiLocation>>, // TODO: Fees?
```

Comment:
Is this the place where we would add which calls are allowed to be free and which ones with fees? If so do you have some example to look at and see how that would look like? I think this is still worked on on the parity side.

Reply:
I tried to look into fees but AFAICT it is indeed still being worked on from the parity side. Not a lot there yet